cli/src/main/java/org/jboss/galleon/cli/HelpSupport.java

```diff
@@ -170,7 +171,7 @@ private static String pad(int max, String k) {
     private static String getHelp(String path) throws IOException {
         InputStream helpInput = CliMain.class.getResourceAsStream(path);
         if (helpInput != null) {
-            try (BufferedReader reader = new BufferedReader(new InputStreamReader(helpInput))) {
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(helpInput, StandardCharsets.UTF_8.name()))) {
```

Comment:
Perhaps use the Charset instead of the name, some places there is name, others instance.

Reply:
ah, good catch, thought there is only InputStreamReader(InputStream in, String charsetName) and missed InputStreamReader(InputStream in, Charset cs) :)